topic1/ejercicio4/ejercicio9.js

```diff
@@ -0,0 +1,48 @@
+function createCORSRequest(method, url) {
```

Comment:
is better to expect just an object with multiple keys like `{ method: 'GET', url: 'some/path'}`